src/library/scala/sys/process/Process.scala

```diff
@@ -172,6 +174,50 @@ trait ProcessCreation {
     require(files.nonEmpty)
     files.map(_.cat).reduceLeft(_ #&& _)
   }
+
+  /**
+    * Echo the given text.
+    *
+    * This can be used as the source of input for a process, for example:
+    *
+    * {{{
+    * import scala.sys.process._
+    * import scala.sys.process.Process.echo
+    *
+    * echo("Hello, world!") #> "curl -d - https://example.com/upload" !
```

Comment:
Postfix syntax now requires importing `scala.language.postfixOps`. I suggest the following:

```suggestion
    * echo("Hello, world!") #> ("curl -d - https://example.com/upload").!
```

Reply:
You'd need parens. I think one must suck it up, so to speak. I don't know if they intend to eliminate postfix, if the language import is just the methadone of syntax and they're actually going to cut us off, but this has to be the poster child for keeping us hooked.